pyiso/caiso.py

```diff
@@ -283,7 +284,24 @@ def get_AS_dataframe(self, anc_region, latest=True, start_at=False, end_at=False
         MW columns holds $/MW float.
         If no data, returns an empty dataframe.
         """
-        # set args
+        # set args, handle latest differently than LMP b/c daily publication schedule
```

Comment:
It could return a 1-hour (+ 1 second) window.  That would guarantee some return and might be more consistent with the other functions.